ui/app/AppLayouts/Chat/ContactsColumn.qml

```diff
@@ -418,6 +418,114 @@ Item {
                         anchors.rightMargin: Theme.padding
                         color: Theme.blue
                         visible: unviewedMessagesCount > 0
+                        anchors.top: applicationWindow.top
+                        anchors.topMargin: 0
+                        anchors.left: parent.left
+                        anchors.leftMargin: Theme.padding
+                        radius: 8
+                        // Hide the box if it is filtered out
+                        property bool isVisible: searchStr == "" || name.includes(searchStr)
+                        visible: isVisible ? true : false
+                        height: isVisible ? 64 : 0
```

Comment:
Lines from 421 to 429 can be removed I think. They are copied lines from the container